utils/lib/__tests__/Quickstart.test.js

```diff
@@ -100,6 +100,37 @@ describe('Quickstart', () => {
 
       expect(qs.isValid).toBe(false);
     });
+
+    test('Ensure quickstart is invalid from an invalid Alert', () => {
+      jest.spyOn(global.console, 'error').mockImplementation(() => {});
+      jest.spyOn(global.console, 'log').mockImplementation(() => {});
+      const qs = new Quickstart(
+        'quickstarts/mock-quickstart-6/config.yml',
+        MOCK_FILES_BASEPATH
+      );
+
+      const components = qs.getComponents();
+
+      expect(components).toBeDefined();
+      expect(qs.isValid).toBe(true);
+
+      // all components are invalid
+      const invalidComponents = components.map((component) => {
+        if (!component.isValid) {
+          return component;
+        }
+      }).filter(Boolean);
+
+      qs.validate();
+
+      expect(invalidComponents).toHaveLength(1)
+
+      const invalidComponent = invalidComponents.pop();
+      expect(invalidComponent.identifier).toBe('mock-alert-policy-3')
```

Comment:
Okay so this is me being totally unreasonably critical and I well welcome you telling me this is too nitpicky, but I'd advocate for us avoiding `pop` and using `[0]` (optionally chained, if needed) inline instead. Reason being is that it alters the shape of `invalidComponents` variable which could be an issue if we decide to make further assertions on that variable.

Reply:
Don't thank thats unreasonable at all! I'll update that